sunkit_magex/pfss/tests/test_fieldline.py

```diff
@@ -27,10 +27,10 @@ def test_open(x, open, pol):
 
 
 @pytest.mark.parametrize(("x", "cls"),
-                         [[[1, 2.5], ClosedFieldLines],
-                          [[1, 1], OpenFieldLines],
+                         [([1, 2.5], ClosedFieldLines),
+                          ([1, 1], OpenFieldLines),
                           ])
 def test_flines_errors(x, cls):
     fline = FieldLine(x, [0, 0], [0, 0], None)
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="goose"):
```

Comment:
erm wat 🪿

Reply:
I use it to check the message before I fix the test.